demos/tests/cases.py

```diff
@@ -254,7 +261,11 @@ def single_option_cases(key, *args):
 ]
 
 PYTHON_DEMOS = [
-    # TODO: 3d_segmentation_demo: no input data
+    PythonDemo(subdirectory='3d_segmentation_demo', device_keys=['-d'], test_cases=combine_cases(
+        TestCase(options={'-m': ModelArg('brain-tumor-segmentation-0001'),
+                          '-o': '.'}),
+        single_option_cases('-i', *IMAGE_SEQUENCES['brain-tumor-nifti']),
```

Comment:
Yes, and for most other models we use several inputs for testing.

Reply:
(assuming you mean "demos", rather than "models")

That's because other demos _take_ multiple inputs, so giving them multiple inputs is cheap. This demo only takes a single input, so the only way to test it with multiple inputs is to run it multiple times, which is going to be slow (especially on the GPU). Which is why I think a justification for using multiple inputs is required in this case.

Reply:
**Demos**, of course, thanks!

I could make only one *image* for testing, but it wouldn't be appropriate to validate demo with only one infer (and we will do this in future). I dont think, that 5 runs is much more time-consuming than 1 in overall test time. But if you are sure about these points and ready to take responsibility , I'll agree with you.

Reply:
fixed